src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -73,13 +89,107 @@ public ProjectInfo ResolveInput(StrykerOptions options)
                 {
                     throw new DirectoryNotFoundException($"Can't find {folder}");
                 }
+
                 inputFiles.Add(FindInputFiles(folder, projectUnderTestDir));
             }
-            result.ProjectContents = inputFiles;
 
-            ValidateResult(result, options);
+            return inputFiles;
+        }
 
-            return result;
+        private FolderComposite FindProjectFilesUsingBuildAlyzer(ProjectInfo info)
```

Comment:
Why not pass the ProjectUnderTestAnalyzerResult instead of the whole project info?

Reply:
redesigned to use ...result instead of the all project info struct